alf/examples/ppo_cart_pole_conf.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2021 Horizon Robotics and ALF Contributors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import alf
+from alf.examples import ppo_conf
+from alf.algorithms.data_transformer import RewardScaling
+from alf.networks import ActorDistributionNetwork, ValueNetwork
+from alf.utils.losses import element_wise_huber_loss
+
+# Environment Configuration
+alf.config(
+    'create_environment', env_name='CartPole-v0', num_parallel_environments=8)
+
+# Reward Scailing
+alf.config('TrainerConfig', data_transformer_ctor=RewardScaling)
+alf.config('RewardScaling', scale=0.01)
+
+# algorithm config
+alf.config('ActorDistributionNetwork', fc_layer_params=(100, ))
+alf.config('ValueNetwork', fc_layer_params=(100, ))
+
+alf.config(
+    'ActorCriticAlgorithm',
+    actor_network_ctor=ActorDistributionNetwork,
+    value_network_ctor=ValueNetwork,
+    optimizer=alf.optimizers.Adam(lr=1e-3))
+
+alf.config(
+    'PPOLoss',
```

Comment:
Now PPOLoss is not used by PPOAlgorithm by default. 

https://github.com/HorizonRobotics/alf/blob/pytorch/alf/algorithms/ppo_algorithm.py#L34

Reply:
Probably should fix PPOAlgorithm so that by default it uses PPOLoss.